CCP_App/src/main/java/com/example/demo2/custom/exception/RequiredInputNotSuppliedException.java

```diff
@@ -0,0 +1,23 @@
+/************************************************************************
+ *                                                                      *
+ *  DDDD     SSSS    AAA        Daten- und Systemtechnik Aachen GmbH    *
+ *  D   D   SS      A   A       Pascalstrasse 28                        *
+ *  D   D    SSS    AAAAA       52076 Aachen-Oberforstbach, Germany     *
+ *  D   D      SS   A   A       Telefon: +49 (0)2408 / 9492-0           *
+ *  DDDD    SSSS    A   A       Telefax: +49 (0)2408 / 9492-92          *
+ *                                                                      *
+ *                                                                      *
+ *  (c) Copyright by DSA - all rights reserved                          *
+ *                                                                      *
+ ************************************************************************
+ *
+ * Author : Riyaardh Adam
+ * Created: 2023/02/21 10:31
+ *
+ ***********************************************************************/
+package com.example.demo2.custom.exception;
```

Comment:
Please rework your packages.

you have ....custom.exception
and .exception.handler

These can be merged into a package and subpackages